src/compas_fab/robots/constraints.py

```diff
@@ -155,24 +211,24 @@ class JointConstraint(Constraint):
 
     Attributes
     ----------
-    joint_name: string
+    joint_name : :class:`str`
         The name of the joint this contraint refers to.
-    value: float
+    value : :class:`float`
         The targeted value for that joint.
-    tolerance_above: float
+    tolerance_above : :class:`float`
         Tolerance above the targeted joint value, in radians. Defaults to 0.
```

Comment:
I think the 0 here and a couple lines below should be surrounded by ticks.

Reply:
c43fb9b